src/shell/autoexec.cpp

```diff
@@ -619,6 +628,54 @@ std::string AutoExecModule::BuildAutoMountFolderCommand(
 	return command;
 }
 
+std::vector<std_fs::path> AutoExecModule::GetMountableImagesPaths(const std_fs::path& drive_path)
+{
+	std::vector<std_fs::path> image_paths;
+
+	for (auto const& entry : std_fs::directory_iterator(drive_path)) {
+		if (entry.exists() && entry.is_regular_file() ) {
+			auto const& path = entry.path();
+			std::string ext = path.extension();
+			std::ranges::transform(ext, ext.begin(), ::tolower);
+			if (ext == ".iso" || ext == ".cue") {
+				image_paths.push_back(path);
+			}
+		}
+	}
+
+	return image_paths;
+}
+
+std::string AutoExecModule::BuildAutoMountImagesCommand(
```

Comment:
`BuildAutoMountCdImagesCommand` perhaps? We can refactor later if we decide to generalise this.